sound/soc/codecs/hdac_hdmi.c

```diff
@@ -123,6 +123,8 @@ struct hdac_hdmi_dai_port_map {
 
 struct hdac_hdmi_drv_data {
 	unsigned int vendor_nid;
+	int *port_map;
```

Comment:
comment here, what is port_map ?

Reply:
i would move the static table just above this to help the reviewer connect the dots

Reply:
const int *port_map?